resources/stats/stats.sh

```diff
@@ -15,20 +15,47 @@ cd "$THIS_SCRIPT_PATH"
 
 platforms=(android/ ios/ linux/ mac/ web/ windows/ developer/ core/ common/models/)
 
-startdate=2022-05-01
-stable=stable-15.0
+builder_describe "Stats collector for Keyman PRs and issues" \
+  stats+ \
+  "--start-date,-d=startDate       Start date for collecting data, yyyy-mm-dd" \
+  "--start-sprint,-s=startSprint   Or, first sprint to start collecting data for" \
+  "--stable,-S=stableBranch        Branch name for previous stable release"
```

Comment:
Is there a way to note --stable is a required parameter, along with either --start-date or --start-sprint?

Reply:
I don't think we do that in builder scripts at present. We could add it to the description line?

(I don't want to do any changes to the builder scripts here, and I think until we actually need required options for other build scripts it's not worth changing builder for this little utility script.)

Reply:
```suggestion
  "--stable,-S=stableBranch        Branch name for previous stable release (required)"
```